Tabloid/client/src/components/users/UserProfiles.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import { getAllUsers } from "../../Managers/UserProfileManager";
+import { UserProfileItem } from "./UserProfileItem";
+import { Table } from "reactstrap";
+
+
+export default function UserProfiles (){
+
+  const [userProfiles, setUserProfiles] = useState([]);
+
+  const getUserProfiles = () =>{
+    getAllUsers().then(allUsers => setUserProfiles(allUsers))
+  }
+  
+  useEffect(() => {
+        getUserProfiles();
+    }, []); 
+  
+  return (
+    <Table>
```

Comment:
`<Table></Table>` can be wrapped with `<div className="m-5"></div>`

This is the bootstrap class property outlined in the Google Doc and provides the margin of 5 all around.

See the code in `PostList.js` for an example.